labs/easydoser.md

```diff
@@ -0,0 +1,35 @@
+# Lab Name
+EasyDoser
+
+# Short Description
+Ease endorsement operations for Hyperledger based products. In other words, provide easy ways to read/modify endorsement policies for Hyperledger based systems.
+
+# Scope of Lab
+Endorsement policy controls the approval of a proposal to be executed at various granular levels.
+In other words, this is the primary entity which is validated before a transaction is committed.
+Endorsement policy can operate at two different granularities:
+
+_Namespace level_: They can be set for the whole namespace.
+This is the default approach in which endorsement policies are specified in the chaincode definition, which is agreed to by channel members and then committed to a channel
+
+_Individual key level_: They can be set for individual state keys called private data collections. You can also specify an endorsement policy at the private data collection level, which would override the chaincode level endorsement policy for any keys in the private data collection.
+This would further restrict which organizations can write to a private data collection.
+
+## EasyDoser architecture
+![alt text](https://cacoo.com/diagrams/3M6r1235YWmvGCyb-1ABD5.png)
+
+# Initial Committers
```

Comment:
Small nitpick, can you move the initial committers a few positions down so that they are in the same order as the template ? (Makes it easier to scan / parse the document later)